Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -397,9 +359,12 @@ - (void)drawInContext:(CGContextRef)cgContext
             ++m_countOfTransactionsWithNonEmptyLayerChanges;
 
         if (m_remoteLayerTreeHost->updateLayerTree(connection, layerTreeTransaction)) {
-            if (!m_replyForUnhidingContent)
+            if (!m_replyForUnhidingContent && !m_activityStateChangeForUnhidingContent) {
```

Comment:
Please make sure we don't get stuck if the process crashes and IDs reset (I think this one might originate in the UIP and thus be OK?)